testing/constraints-3.6.txt

```
@@ -0,0 +1,8 @@
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
+# List *all* library dependencies and extras in this file.
+# Pin the version to the lower bound.
+#
+# e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
+# Then this file should have foo==1.14.0
+google-api-core==1.22.0
```

Also [proto-plus](https://github.com/googleapis/python-dialogflow/blob/master/setup.py#L24) 
```suggestion
google-api-core==1.22.0
proto-plus==1.10.0
```

---

Thanks!